product-service/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+const slsw = require('serverless-webpack');
+
+module.exports = {
+  entry: slsw.lib.entries,
+  mode: 'production',
```

Comment:
it's good to distinguish modes: slsw.lib.webpack.isLocal ? 'development' : 'production'